examples/mockups/index.html

```diff
@@ -0,0 +1,329 @@
+                </p>
+                <p>
+                  <a href="https://blog.readthedocs.com/newsletter-september-2021/">Read more</a>
+                </p>
+              </div>
+            </div>
+            <div class="column">
+              <div class="ui basic segment">
+                <div><span class="ui label">Update</span></div>
+                <h4>You are all up to date</h4>
+                <p>
+                  We post important announcements, news and updates in this area. You can also check out our <a href="https://blog.readthedocs.com/">blog</a> or subscribe the newsletter to receive future updates first hand, plus receive our monthly report.
+                </p>
+                <p>
+                  <a href="#">Subscribe our newsletter</a>
+                </p>
+              </div>
+            </div>
+
+          </div>
+        </div>
+        
+      </div>
+
+    </section>
+
+    
+    <!-- Infographics -->
+    <section class="ui vertical segment light padded" id="infographics">
+
+      <div class="ui container">
+
+        <div class="ui grid centered">
+          <div class="row">
+            <div class="column fourteen wide computer sixteen wide tablet">
+          
+              <h2 class="ui header">How it works</h2>
+
+              <div class="ui grid stackable three column">
+
+                <div class="column center aligned">
+                  <div class="ui image">
+                    <img src="images/infographic-01.png">
+                  </div>
+                  <p>
+                    Write your documentation easily, using .md or .rst, directly in your own IDE, as you work on what maters: your project.
+                  </p>
+                </div>
+                <div class="column center aligned">
+                  <div class="ui image">
+                    <img src="images/infographic-02.png">
+                  </div>
+                  <p>
+                    Upload your work to your favorite source code repository hosting service. We support GitHub, GitLab, BitBucket and more.
+                  </p>
+                </div>
+                <div class="column center aligned">
+                  <div class="ui image">
+                    <img src="images/infographic-03.png">
+                  </div>
+                  <p>
+                    Setup your account with Read the Docs, add a .config file to tell us your preferences and we'll do the rest!
+                  </p>
+                </div>
+
+              </div>
+
+            </div>
+          </div>
+        </div>
+
+      </div>
+
+    </section>
+
+
+    <!-- It works -->
+    <section class="ui vertical segment very padded" id="it-works">
+
+      <div class="ui container">
+        <div class="ui grid center aligned">
+
+          <div class="row">
+            <div class="column twelve wide computer sixteen wide tablet">
+          
+              <p class="shout text">
```

Comment:
Makes sense to align the verbiage.
It is a custom class. I could only find SUI native size styling when applied to components (e.g. buttons, labels) not to simple text and I think that we will want to have the ability to size simple text with classes (e.g. spans, paragraphs), not only for this specific place but also for a more generic purpose.

I started by using a different verbiage to avoid class conflicts with SUI... but since I'm using both words for the style anyway - `.shout.text` - it makes more sense to use SUI's wording. I'll change this.

Reply:
This is where you normally specify the size of the surrounding element, like:

```
<div class='ui basic huge segment'><p>...</p></div>
```

This covers all elements in the segment, not just individual text elements.

Reply:
Done. Removed the class and used a `<div class='ui basic huge segment'><p></p></div>` instead.

I feel we are missing a lot of polish being limited only to the native options as we loose much fine control over spacings,  line-height and font weight.